server/game/cards/01_SOR/units/BlackOneScourgeOfStarkillerBase.ts

```diff
@@ -0,0 +1,29 @@
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import AbilityHelper from '../../../AbilityHelper';
+
+export default class BlackOneScourgeOfStarkillerBase extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8429598559',
+            internalName: 'black-one#scourge-of-starkiller-base'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'You may discard your hand',
```

Comment:
nit: our convention is to omit the "you may" since that's implied by the presence of the pass button and just state the ability effect